### PR TITLE
Mobile version completed

### PR DESCRIPTION
**Pull request by shakespear95**
## Summary
- added the rest of the mobile version
- includes the following sections
    - Portfolio
    - Languages
    - Frameworks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
...
+    <br>
+    <section class="tonic">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
+        </div>
+        <ul class="toniclists1">
+            <li class="canopy">
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly take note that you are to wrap all the cards section in `main` tag and display it as grid. 😄 
crr5

File: index.html [hunk 2]
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
...
     </section> 
+    <br>
+    <br>
+    <br>
+    <section class="tonic">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly ensure that all your cards' titles use header tag `hx` not `p` tag. 👍

File: index.html [hunk 3]
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
...
+            <div class="text2">
+                <p>
+                    I am an Information and Communication Engineer and full stack website developer. I can help you build a product, feature or website look throug some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. 
+                </p>
+            </div>
+           <br>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <p>LET'S CONNECT </p>
```
Comment (OyePriscilla) [inline index.html]: - [ ] Kindly use header tag as well for this title. 😄

File: index.html [hunk 4]
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
...
+                </div>
+                <br>
+                <div class="css">
+                    <div>
+                        <img src="img/Ellipse 1 (3).png" alt="css">
+                    </div>
+                    <div class="csstext">
+                        <p>CSS</p>
+                    </div>
+                </div>
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly ensure that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. 😄 
- [x] It is better to align them according to the template. 👍 

## Your Design 

crr6


## Figma Design Template

crr7

File: index.html [hunk 5]
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
...
+            <img src="img/Separator Bottom.png" alt="separater">
+
+            <div class="skillz">
+                <div class="skillstext">
+                    <p>Skills</p>
+                </div>
+                <div class="chevron">
+                    <img src="img/chevron-right.svg" alt="vector">
+                </div>
+            </div>
```
Comment (OyePriscilla) [inline index.html]: - [x] Please take note that the technology categories ("Languages", "Frameworks" and "Skills") are also in a `ul`, each of them in a `li`. 😄 
- [x] I noticed that the Framework category does not have chevron-right icon. Kindly add it. 👍

File: css/style.css [hunk 6]
```diff
@@ -96,3 +96,399 @@ nav {
...
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+.cardbut {
+  box-sizing: border-box;
+  flex-direction: column;
```
Comment (OyePriscilla) [inline css/style.css:468]: - [x] Kindly implement the buttons styling for hover, active and pressed. 👍 

## Figma Design Template
 
crr8

- [x] Also, all your buttons are not properly aligned with other content; you may have to adjust the margin-left. 👍 


crr9

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
## Hi @shakespear95 
Nice job, you are almost there.


 
## (Highlights) Good Points: 
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

## (Changes Required) 

* [x]  Please check my inline comments for other changes required.
 
## [Optional] suggestions:

- [x] [optional] Please take note that commit messages should be in sentence case and be well descriptive. It would be great if you start implementing this with subsequent PRs.

> Having a good commit message history will allow you (or other developers working on your code) to quickly see what changes were made and why. This is useful if a bug is found in the code that needs to be fixed!


 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
**Cheers and Happy coding! :clap::clap::clap:**
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @oyepriscilla in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 5 above)
Comment (shakespear95) [reply index.html]: Thank you for noticing this error, I fixed it by adding another chevron-right icon. Issue fixed

File: index.html (hunk 4 above)
Comment (shakespear95) [reply index.html]: Thank you for noticing this part, I had put too much space between the images and the text.
to fix it I had to reduce the margin that I had put in between the images and the text. 
thank you again.

File: css/style.css (hunk 6 above)
Comment (shakespear95) [reply css/style.css:468]: ## Resolved 👍 
- added the effects
- effects added
    - on hover
    - on press
    - active
- thanks for noticing

Jack Sparrow Salute

File: index.html (hunk 1 above)
Comment (shakespear95) [reply index.html]: ## resolved 
- I did not follow instructions to fix everything in the grid 
- I had to remove the <br> and use the grid for everything
- Issue resolved and it's even looking better 
- Thanks again

**Comment by shakespear95** [comment]
summary of the corrections

- I corrected paragraph tag to headings  in the HTML files 
- I corrected the alignment of the pictures and text on the part of the languages
- added the chevron that I had missed
- used the grid instead of <be>
- changed my buttons to respond to hover, press, and active
- thanks for the review

File: index.html (hunk 3 above)
Comment (Kossi-stack) [reply index.html]: - [x] Kindly change the p tag by a header tag

File: index.html (hunk 4 above)
Comment (Kossi-stack) [reply index.html]: - [x] Kindly wrap all technology in a ul list and each of them in li

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @shakespear95,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions
- [OPTIONAL] Kindly use a proper naming for your branches.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by shakespear95** [comment]
-errors fixed 

1. changing the paragraph tag to h2 
2. removing divs on lists and using un ordered lists

File: index.html [hunk 7]
```diff
@@ -69,5 +69,310 @@ <h2>I'm Shakespear Glad to see you!</h2>
...
+                <p>
+                    I am an Information and Communication Engineer and full stack website developer. I can help you build a product, feature or website look throug some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. 
+                </p>
+            </div>
+           <br>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <h1>LET'S CONNECT </h1>
+                </div>
```
Comment (victorokoroji) [inline index.html]: - [x] As the previous reviewer suggested, kindly wrap your `LET'S CONNECT` text in an anchor tag as the previous reviewer has suggested 👍🏽 as the link should take you to the contact page or another page 👍🏽 

e.g.
```
<div class="text3">
         <a href="#">LET'S CONNECT </a>
   </div>
```

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @shakespear95, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @shakespear95, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**